modules/configuring-layer-two-switched-topology.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: CONCEPT
+[id="configuration-layer-two-switched-topology_{context}"]
+= Configuration for a switched topology
+
+The switched (layer 2) topology networks interconnect the workloads through a cluster-wide logical switch. This configuration can be used for IPv6 and dual-stack deployments.
+
+[NOTE]
+====
+Layer 2 switched topology networks only allow for the transfer of data packets between pods within a cluster.
+====
+
+The following `NetworkAttachmentDefinition` custom resource definition (CRD) YAML describes the fields needed to configure a switched secondary network.
+
+[source,yaml]
+----
+    {
+            "cniVersion": "0.3.1",
+            "name": "ns1-l2-network",
+            "type": "ovn-k8s-cni-overlay",
+            "topology":"layer2",
+            "subnets": "10.100.200.0/24",
```

Comment:
For `"topology":"layer2",`, 4.13 code support dual stack "subnets": "192.168.100.0/24,fd00:dead:beef::0/64", should we mention this in the doc?

Reply:
Added to the config object section. Please take a look. Thanks so much!